templates/zerver/integrations/errbot.md

```diff
@@ -25,23 +27,42 @@ Run your favorite chatbot in Zulip!
         BOT_LOG_FILE = r'<path/to/your/errbot/logfile.log>'
         BOT_LOG_LEVEL = logging.INFO
 
-        BOT_IDENTITY = {  # Fill this with the corresponding values in your bot's `.zuliprc`
-          'email': '<err-bot@your.zulip.server>',
+        BOT_IDENTITY = {
+          'email': '<your-bot-name@yourZulipDomain.zulipchat.com>',
           'key': '<abcdefghijklmnopqrstuvwxyz123456>',
-          'site': '<http://your.zulip.server>'
+          'site': '<https://yourZulipDomain.zulipchat.com>'
         }
-        BOT_ADMINS = ('<your@email.address',)
+        BOT_ADMINS = ('<your@email.address>',)
         CHATROOM_PRESENCE = ()
-        BOT_PREFIX = '<@**err-bot**>'  # Providing errbot's full name in Zulip lets it respond to @-mentions.
+        BOT_PREFIX = '@**<your-bot-name>**'
+
+    !!! tip ""
+
+        Access and copy your bot's info from the
+        [bot settings page]({{ zulip_url }}/#settings/your-bots).
 
-    Sections you need to edit are marked with `<>`.
+    Sections you need to edit are marked with `<>`. Replace the `<...>`
+    placeholders with your own values, removing the `<` and `>` brackets.
```

Comment:
Although I was recommended to remove the angular brackets, I think that could get confusing too.
The user might not be able to differentiate between what needs to be replaced and what's not.
So, to clarify that the angular brackets are not part of the syntax itself, I've elaborated this instruction.

Reply:
I think Tim's comment was about using the `zulip_url` variable for the site information, which would have been correct for a specific Zulip organization's documentation. I think the expansion here is fine though.